main.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+	"os"
+
+	"github.com/AlexeyVilkov/go_final_project/database"
+	"github.com/AlexeyVilkov/go_final_project/handlers"
+
+	_ "github.com/mattn/go-sqlite3"
+)
+
+func main() {
+
+	// подключаемся к БД
+	_, err := database.ConnectDB()
+	if err != nil {
+		fmt.Println(err)
```

Comment:
Лучше использовать Fatal чтобы программы вышла с кодом ошибки
```suggestion
		log.Fatal(err)
```